VideoLocker/src/main/java/org/edx/mobile/util/SoftKeyboardUtil.java

```diff
@@ -21,4 +23,21 @@ public static void hide(@NonNull final Activity activity) {
             iManager.hideSoftInputFromWindow(view.getWindowToken(), 0);
         }
     }
+
+    /**
+     * Hides the soft keyboard by clearing EditText's focus in landscape mode.
+     *
+     * @param editText The EditText whose focus needs to be cleared.
+     */
+    public static void clearEditTextFocusInLandscape(@NonNull final EditText editText) {
+        final Context context = editText.getContext();
+        if (context.getResources().getConfiguration().orientation == Configuration.ORIENTATION_LANDSCAPE) {
+            editText.post(new Runnable() {
+                @Override
+                public void run() {
+                    editText.clearFocus();
```

Comment:
I think there should be one utility method for just posting the clear focus action `Runnable` one the event queue (since this could be useful in other cases as well, e.g. `CourseDiscussionTopicsFragment` and `CourseDiscussionPostsSearchFragment`), and maybe another method that calls it conditional on the screen being in landscape mode (or you could just apply the check in the screens separately, as it doesn't really add that much verbosity).